manifests/claudie/dynamo/dynamodb.yaml

```diff
@@ -3,16 +3,19 @@ kind: Deployment
 metadata:
   name: dynamodb
   labels:
-    app: dynamodb
+    app.kubernetes.io/part-of: dynamo
```

Comment:
This is incorrect.
Everything is part-of `claudie`
Same for all other occurrences, please.